peerdid/peer_did.py

```diff
@@ -26,21 +32,50 @@ def create_peer_did_numalgo_0(inception_key: PublicKeyAuthentication) -> PEER_DI
     :param inception_key: the key that creates the DID and authenticates when exchanging it with the first peer
     :return: generated peer_did
     """
-    pass
+    peer_did = 'did:peer:0z' + inception_key.encoded_value
+    if is_peer_did(peer_did):
+        return peer_did
+    else:
+        raise ValueError("Wrong Peer DID")
 
 
 def create_peer_did_numalgo_2(encryption_keys: List[PublicKeyAgreement], signing_keys: List[PublicKeyAuthentication],
-                              service_endpoint: str) -> PEER_DID:
+                              service: JSON) -> PEER_DID:
     """
     Generates peer_did according to the second algorithm
     (https://identity.foundation/peer-did-method-spec/index.html#generation-method)
     For this type of algorithm did_doc can be obtained from peer_did
     :param encryption_keys: list of encryption keys
     :param signing_keys: list of signing keys
-    :param service_endpoint: JSON string conforming to the DID specification (https://www.w3.org/TR/did-core/#services)
+    :param service: JSON string conforming to the DID specification (https://www.w3.org/TR/did-core/#services)
     :return: generated peer_did
     """
-    pass
+    encryption_keys_str = '.E' + ''.join(key.encoded_value for key in encryption_keys)
+    signing_keys_str = '.V' + ''.join(key.encoded_value for key in signing_keys)
+    service = '.S' + encode_service(service)
+
+    peer_did = 'did:peer:2' + encryption_keys_str + signing_keys_str + service
+    if is_peer_did(peer_did):
+        return peer_did
+    else:
+        raise ValueError("Wrong Peer DID")
+
+
+def encode_service(service: JSON) -> str:
```

Comment:
Please a unit test for this particular method. 
We can take the values from https://identity.foundation/peer-did-method-spec/index.html#multi-key-creation (Example 3):
Input
```
{
	"type": "didcommmessaging",
	"serviceEndpoint": "https://example.com/endpoint",
	"routingKeys": ["did:example:somemediator#somekey"]
}
```
Expected Output: `.SeyJ0IjoiZG0iLCJzIjoiaHR0cHM6Ly9leGFtcGxlLmNvbS9lbmRwb2ludCIsInIiOlsiZGlkOmV4YW1wbGU6c29tZW1lZGlhdG9yI3NvbWVrZXkiXX0=`

We can add more tests that check:
-  how it works when some of service params are missing (such as routing_keys)
-  how it works when the service is not of type `didcommmessaging`, and has additional fields (It looks like we need to define how it should work in this case: still encode, throw an exception (unsupported service type), or just ignore the service). 
- the case when service consists of multiple entries (possibly different types). For example,
```
[
{
	"type": "didcommmessaging",
	"serviceEndpoint": "https://example.com/endpoint",
	"routingKeys": ["did:example:somemediator#somekey"]
},
{
	"type": "didcommmessaging",
	"serviceEndpoint": "https://example.com/endpoint2",
	"routingKeys": ["did:example:somemediator#somekey2"]
}
]
```